chapters/annotations.tex

```diff
@@ -1099,6 +1099,7 @@ \subsubsection{Text}\label{text}
   When quoted identifiers (e.g., \lstinline!rec.'}'.'quoted ident'!) are involved, the form \%\{\emph{par}\} must be used.
   Here, \emph{par} is a general \lstinline[language=grammar]!component-reference!, and the macro can be directly followed by a letter.
   Thus \lstinline!%{w}x%{h}! gives the value of \lstinline!w! directly followed by \emph{x} and the value of \lstinline!h!, while \lstinline!%wxh! gives the value of the parameter \lstinline!wxh!.
+  Additionally, \lstinline!%{a.p}! should give the value of the parameter \lstinline!p! in the component \lstinline!a!.
```

Comment:
Stick to tense:
```suggestion
  Additionally, \lstinline!%{a.p}! gives the value of the parameter \lstinline!p! in the component \lstinline!a!.
```

Reply:
Tense-change was included in rewritten text